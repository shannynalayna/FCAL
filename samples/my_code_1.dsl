/* This program is just a copy of sample_1.dsl. You should write your FCAL program to multiply two
2-dimensional matrices and save it
in a file with this name. */

main () {
	matrix m1 = matrix_read( "./samples/my_code_1_m1.data");
	matrix m2 = matrix_read( "./samples/my_code_1_m2.data");
	matrix result = m1 * m2;
	print(result);
}



