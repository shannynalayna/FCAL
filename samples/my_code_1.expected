3 4
55  70  85  100  
70  90  110  130  
85  110  135  160  
